composer.json

```diff
@@ -3,13 +3,13 @@
   "type": "project",
   "description": "Findologic api plugin for plentymarkets system",
   "require": {
-    "php": "^7.0",
-    "findologic/http_request2": "2.3.1",
+    "php": "^8.0",
     "ext-simplexml": "*",
-    "squizlabs/php_codesniffer": "^3.5"
+    "findologic/http_request2": "2.4",
```

Comment:
https://github.com/findologic/HTTP_Request2/releases

> Establish support for PHP 7.4.

Can you please verify if 2.4. works with php8 as well.

Reply:
I checked the code in the IDE and ran the tests with PHP8.0.
Seems to have no issues.